<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Preferences</class>
 <widget class="QDialog" name="Preferences">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>356</width>
    <height>421</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DarkFlow Preferences</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="tabBarAutoHide">
      <bool>false</bool>
     </property>
     <widget class="QWidget" name="tab_resources">
      <attribute name="title">
       <string>Resources</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>ImageMagick Architecture Settings</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="2" column="1">
           <widget class="QLineEdit" name="valueArea">
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="8" column="2">
           <widget class="QLineEdit" name="defaultThreads">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="8" column="1">
           <widget class="QLineEdit" name="valueThreads">
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="labelDisk">
            <property name="toolTip">
             <string>Maximum amount of disk space in bytes permitted for use by the pixel cache. If this limit is exceeded, the pixel cache is not created and a fatal exception is thrown.</string>
            </property>
            <property name="text">
             <string>Disk:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="labelArea">
            <property name="toolTip">
             <string>Maximum area in bytes of any one image that can reside in the pixel cache memory. If this limit is exceeded, the image is automagically cached to disk and optionally memory-mapped.</string>
            </property>
            <property name="text">
             <string>Area:</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QLineEdit" name="valueDisk">
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="valueMemory">
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="labelSettings">
            <property name="text">
             <string>Your settings</string>
            </property>
           </widget>
          </item>
          <item row="4" column="2">
           <widget class="QLineEdit" name="defaultMap">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="7" column="0" colspan="3">
           <widget class="Line" name="line">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="labelMap">
            <property name="toolTip">
             <string>Maximum amount of memory map in bytes to allocate for the pixel cache.</string>
            </property>
            <property name="text">
             <string>Map:</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QLineEdit" name="valueMap">
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="5" column="2">
           <widget class="QLineEdit" name="defaultDisk">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QLineEdit" name="defaultArea">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="labelMemory">
            <property name="toolTip">
             <string>Maximum amount of memory in bytes to allocate for the pixel cache from the anonymous mapped memory or the heap.</string>
            </property>
            <property name="text">
             <string>Memory:</string>
            </property>
           </widget>
          </item>
          <item row="3" column="2">
           <widget class="QLineEdit" name="defaultMemory">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="8" column="0">
           <widget class="QLabel" name="labelThreads">
            <property name="toolTip">
             <string>Maximum number of threads that are permitted to run in parallel within ImageMagick functions.</string>
            </property>
            <property name="text">
             <string>Threads:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QLabel" name="labelCurrent">
            <property name="text">
             <string>Defaults (GBytes)</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>DarkFlow Architecture Settings</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="1" column="1">
           <widget class="QLineEdit" name="valueDflWorkers">
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QLineEdit" name="defaultDflThreads">
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="labelDflThreads">
            <property name="text">
             <string>Threads:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="valueDflThreads">
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="labelDflWorkers">
            <property name="text">
             <string>Workers:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QLineEdit" name="defaultDflWorkers">
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="labelDflSettings">
            <property name="text">
             <string>Your settings</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLabel" name="labelDflCurrent">
            <property name="text">
             <string>Defaults</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Pixels</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="labelTransformTarget">
         <property name="text">
          <string>Display target:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="comboTransformTarget">
         <item>
          <property name="text">
           <string>None</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>sRGB</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>IUT BT.709</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Square Root</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="3" column="0">
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>298</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="labelIncompatibleScale">
         <property name="text">
          <string>Incompatible scale:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="comboIncompatibleScale">
         <item>
          <property name="text">
           <string>Ignore and convert</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Warning and convert</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Error</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="labelLabSelectionSize">
         <property name="text">
          <string>Lab selection size:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QSpinBox" name="spinLabSelectionSize">
         <property name="suffix">
          <string> pixels</string>
         </property>
         <property name="minimum">
          <number>256</number>
         </property>
         <property name="maximum">
          <number>2000</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_path">
      <attribute name="title">
       <string>Path</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="labelBaseDir">
         <property name="text">
          <string>Base Directory:</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLineEdit" name="valueBaseDir"/>
         </item>
         <item>
          <widget class="QPushButton" name="buttonBaseDir">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="labelTmpDir">
         <property name="text">
          <string>Temporary Directory:</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLineEdit" name="valueTmpDir"/>
         </item>
         <item>
          <widget class="QPushButton" name="buttonTmpDir">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_logging">
      <attribute name="title">
       <string>Logging</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="1" column="1">
        <widget class="QComboBox" name="comboRaiseOn">
         <item>
          <property name="text">
           <string>Debug</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Info</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Warning</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Error</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Critical</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Never raise</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="3" column="0">
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="labelRaiseOn">
         <property name="text">
          <string>Raise console on:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="comboLogLevel">
         <item>
          <property name="text">
           <string>Debug</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Info</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Warning</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Error</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Critical</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="labelLogLevel">
         <property name="text">
          <string>Log Level:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="labelTrapOn">
         <property name="text">
          <string>Trap on (debug):</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QComboBox" name="comboTrapOn">
         <item>
          <property name="text">
           <string>Debug</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Info</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Warning</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Error</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Critical</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Never Trap</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel|QDialogButtonBox::Ok|QDialogButtonBox::Reset</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Preferences</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>257</x>
     <y>411</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Preferences</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>325</x>
     <y>411</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>clicked(QAbstractButton*)</signal>
   <receiver>Preferences</receiver>
   <slot>clicked(QAbstractButton*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>186</x>
     <y>411</y>
    </hint>
    <hint type="destinationlabel">
     <x>177</x>
     <y>154</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBaseDir</sender>
   <signal>clicked()</signal>
   <receiver>Preferences</receiver>
   <slot>baseDirClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>119</x>
     <y>77</y>
    </hint>
    <hint type="destinationlabel">
     <x>177</x>
     <y>154</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonTmpDir</sender>
   <signal>clicked()</signal>
   <receiver>Preferences</receiver>
   <slot>tmpDirClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>119</x>
     <y>85</y>
    </hint>
    <hint type="destinationlabel">
     <x>177</x>
     <y>154</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>clicked(QAbstractButton*)</slot>
  <slot>tmpDirClicked()</slot>
  <slot>baseDirClicked()</slot>
 </slots>
</ui>
